docs/user-guide/orphaned-resources.md

```diff
@@ -0,0 +1,17 @@
+# Orphaned Resources Monitoring
```

Comment:
I have committed some grammar changes to this

Reply:
thank you!